src/Scenes/Scene.cpp

```diff
@@ -371,7 +386,11 @@ void Scene::enter()
             _hudPtr->setBalance(cash);
         });
 		
-    if (_bgmusic) _bgmusic->play();		
+    if (_bgmusic) 
+    {
+        _logger->debug("Playing scene music.");
```

Comment:
This reminds me that I need to do the audio system refactor. I want to rewrite it as a [Service Provider](https://gameprogrammingpatterns.com/service-locator.html) so that way we can easily disable and enable it locally, and also adjust the volume.

Of course, we'd also need settings too.

Anyway, this is no comment about the code, just a comment for my own sanity about the work I'd lke to do.